Makefile

```diff
@@ -112,6 +112,10 @@ terraform/install: | $(BIN_DIR) guard/program/jq
 	$(@D) --version
 	@ echo "[$@]: Completed successfully!"
 
+terragrunt/install: TERRAGRUNT_VERSION ?= latest
+terragrunt/install: | $(BIN_DIR) guard/program/jq
+	@ $(MAKE) install/gh-release/$(@D) FILENAME="$(BIN_DIR)/$(@D)" OWNER=gruntwork-io REPO=$(@D) VERSION=$(TERRAGRUNT_VERSION) QUERY='.name | endswith("$(OS)_$(ARCH)")'
```

Comment:
Perhaps insert before this line:

```
        @ echo "[$@]: Installing $(@D)..."
```

Reply:
That's done already in the target `install/gh-release/%`... for example:

```
$ make terragrunt/install
[install/gh-release/terragrunt]: Installing terragrunt...
curl --fail -sSL -H "Authorization: token $GITHUB_ACCESS_TOKEN" -o /home/loren/bin/terragrunt https://github.com/gruntwork-io/terragrunt/releases/download/v0.29.2/terragrunt_linux_amd64
chmod +x /home/loren/bin/terragrunt
terragrunt --version
terragrunt version v0.29.2
[install/gh-release/terragrunt]: Completed successfully!
```